gcp/table_gcp_disk_metric_read_ops_hourly.go

```diff
@@ -0,0 +1,42 @@
+package gcp
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	"google.golang.org/api/compute/v1"
+)
+
+//// TABLE DEFINITION
+
+func tableGcpDiskMetricReadOpsHourly(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_disk_metric_read_ops_hourly",
+		Description: "GCP Compute Disk Daily connections",
+		List: &plugin.ListConfig{
+			ParentHydrate: listComputeInstances,
+			Hydrate:       listDiskMetricReadOpsHourly,
+		},
+		Columns: monitoringMetricColumns([]*plugin.Column{
+			{
+				Name:        "instance_id",
+				Description: "The VM Instance name.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DimensionValue"),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listDiskMetricReadOpsHourly(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	instanceInfo := h.Item.(*compute.Instance)
```

Comment:
Is it correct?
Why are we fetching `Instance` details, since the table stores `disk` metrics?